itl/plugins-contrib.d/icingacli.conf

```diff
@@ -18,20 +30,34 @@ object CheckCommand "icingacli-businessprocess" {
 			set_if = "$icingacli_businessprocess_details$"
 			description = "Get details for root cause analysis"
 		}
-		"--state-type" = {
-			value = "$icingacli_businessprocess_statetype$"
-			description = "Define which state type to look at. Could be either soft or hard, overrides an eventually configured default"
-		}
+		"--downtime-is-ok" = {
+			set_if = "$icingacli_businessprocess_downtimeisok$"
+			description = "Treat hosts/services in downtime always as UP/OK."
+		}		
 		"--process" = {
 			value = "$icingacli_businessprocess_process$"
 			description = "Business process to monitor"
 			skip_key = true
 			required = true
 			order = -1
 		}
+		"--root-cause" = {
+			set_if = "$icingacli_businessprocess_rootcause$"
+			description = "Used in combination with --blame. Only shows the path of the nodes which are responsible for the state of the business process"
+		}
+		"--state-type" = {
+			value = "$icingacli_businessprocess_statetype$"
+			description = "Define which state type to look at. Could be either soft or hard, overrides an eventually configured default"
+		}
 	}
 
+	vars.icingacli_businessprocess_ackisok = false	
+	vars.icingacli_businessprocess_blame = false
+	vars.icingacli_businessprocess_colors = false
 	vars.icingacli_businessprocess_details = false
+	vars.icingacli_businessprocess_downtimeisok = false
+	vars.icingacli_businessprocess_rootcause = false
+	vars.icingacli_businessprocess_statetype = false
```

Comment:
At least the last line shall not be there.

Reply:
correct, got confused by the whole parameter moving